src/list_of_files.cmake

```diff
@@ -72,7 +72,7 @@ set(fortran_src
   chrtrn.f
   chrtrna.f
   cktaba.f
-  closbf.f
+  closbf.F
```

Comment:
Should probably move the `.F`  down with the rest of the `.F`'s.  There is a `openbf.F` down there.